sentences/es/climate_HassClimateGetTemperature.yaml

```
@@ -2,4 +2,9 @@ language: es
 intents:
   HassClimateGetTemperature:
     data:
-    - sentences: []
+      - sentences:
+        - "¿cuál es [la] <temp>?"
+        - "¿qué <temp> hace?"
+        - "¿cuál es [la] <temp> <area>?"
+        - "¿qué <temp> hace en <area>?"
```

We should not include any punctuation in matchers.
```suggestion
        - "cuál es [la] <temp>"
        - "qué <temp> hace"
        - "cuál es [la] <temp> <area>"
        - "qué <temp> hace en <area>"
```